platform/podjs/README.md

```diff
@@ -29,12 +29,102 @@ soon.
 
 Simply run `build.js` in the repository root.
```

Comment:
Not part of the PR, but... Either run `./build.js` or `node build.js`. Also it needs to be version >= 16 and npm version >= 7 for the time being.

Reply:
WRT to the text above, as long as you're documenting this
* Link to definitions of things such as polyPod. (line 3)
* `polyOut` and `polyNav`  (line 23) were implemented in #59